lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -3,12 +3,45 @@
                    [multi]="multi">
         <mat-expansion-panel *ngIf="displayDefaultProperties"
                              [expanded]="canExpandProperties()"
-                             [attr.data-automation-id]="'adf-metadata-group-properties'">
+                             [attr.data-automation-id]="'adf-metadata-group-properties'"
+                             hideToggle
+                             (opened)="handleGeneralInfoPanelState()"
+                             (closed)="handleGeneralInfoPanelState()">
             <mat-expansion-panel-header>
-                <mat-panel-title class="adf-metadata-properties-title">
-                    {{ 'CORE.METADATA.BASIC.HEADER' | translate }}
-                </mat-panel-title>
+                <div class="adf-metadata-properties-panel-content">
+                    <mat-icon>
+                        {{ generalInfoPanelState ? 'expand_more' : 'chevron_right'}}
+                    </mat-icon>
+                    <mat-panel-title class="adf-metadata-properties-title">
+                        {{ 'CORE.METADATA.BASIC.HEADER' | translate }}
+                    </mat-panel-title>
+                </div>
+                <button *ngIf="!editable && !readOnly && hasAllowableOperations"
```

Comment:
extract to a separate property to simplify testing and maintenance

```ts
get canToggleEdit(): boolean {
  return !editable && !readOnly && hasAllowableOperations;
}
```

and then

```html
<button *ngIf="canToggleEdit">
```